plugin.video.aswim/addon.xml

```diff
@@ -21,7 +21,7 @@
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
     <website>www.adultswim.com</website>
     <source>https://github.com/learningit/plugin.video.aswim</source>
```

Comment:
This repo doesn't exist.

Reply:
@learningit how would you like me to proceed? I think a good solution would be to alter the string to point to this repo but I don't want to change this if you're still planning to maintain the current location.

Reply:
Please see comments below